src/test/testing-library/ForWhoSection.test.tsx

```
@@ -13,11 +13,13 @@ describe('ForWhoSection component', () => {
 
-  it('should have a proper alt tag', () => {
+  it('should have the correct number of images with empty alt text and images with proper alt text', () => {
     const { getAllByAltText } = render(<ForWhoSection />);
-    const images: HTMLElement[] = getAllByAltText('Vector');
 
-    expect(images).toHaveLength(5);
-    expect(images[0]).toBeInTheDocument();
+    const imagesWithEmptyAlt = getAllByAltText('');
+    expect(imagesWithEmptyAlt).toHaveLength(9);
+
+    const imagesWithAltText = getAllByAltText('Vector');
```

ESLint

[error] Expected imagesWithAltText to have a type annotation.

(@typescript-eslint/typedef)